### PR TITLE
Update checkpoint url

### PR DESCRIPTION
**Pull request by martinpgrayson**
This updates the checkpoint URL to one that is now able to download the checkpoint from blob storage.
I have also updated the MACs stats. We were previously just outputting the details from the last task. This update collects the MACs count after each user, then displays the average and std at the end. I also added that it is a leader board stat too.

### CODE REVIEW

File: orbit_challenge_getting_started.ipynb [hunk 1]
```diff
@@ -257,6 +259,7 @@
     "        if (step+1) % data_queue.num_tasks == 0:\n",
     "            evaluator.set_current_user(task[\"task_id\"])\n",
     "            _,_,_,current_video_stats = evaluator.get_mean_stats(current_user=True)\n",
+    "            macs_to_personalise.append(np.mean(evaluator.macs_counter))\n",
```
Comment (Slowika) [inline orbit_challenge_getting_started.ipynb:262]: We were already reporting the running average.

File: orbit_challenge_getting_started.ipynb [hunk 2]
```diff
@@ -272,7 +275,8 @@
     "print(f\"Average over all objects: {get_stats_str(stats_per_obj)}\")\n",
     "print(f\"Average over all tasks: {get_stats_str(stats_per_task)}\")\n",
     "print(f\"Average over all videos (leaderboard metric): {get_stats_str(stats_per_video)}\")\n",
-    "print(evaluator.get_ops_counter_mean_stats())\n",
+    "mean_macs, std_macs = clever_format([np.mean(macs_to_personalise), np.std(macs_to_personalise)], \"%.2f\")\n",
```
Comment (Slowika) [inline orbit_challenge_getting_started.ipynb:278]: See my comment above. When the loop finishes, the running average returned by `np.mean(evaluator.macs_counter)` is equal to the total mean reported by `evaluator.get_ops_counter_mean_stats()`.

**Comment by Slowika** [comment]
@martinpgrayson @mgrayson-msft It looks good now! :) Thank you for the updates.

**Review by Slowika — APPROVED** [review]

**Comment by martinpgrayson** [comment]
> @martinpgrayson please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Pull request merged**